src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,33 @@
+package calculator
+
+class Calculator {
+    fun calculate(expression: String?): Int {
+        validate(expression)
+
+        val elements = expression!!.split(" ").groupBy { isNumeric(it) }
+        val numbers = elements[true]?.map { it.toInt() }
+        val operators = elements[false]
+
+        if (numbers.isNullOrEmpty()) {
+            return 0
+        }
+
+        if (operators.isNullOrEmpty()) {
+            return numbers[0]
+        }
+
+        return numbers.reduceIndexed { index, total, number ->
+            calculate(total, number, operators[index - 1])
+        }
+    }
+
+    private fun validate(expression: String?) {
+        if (expression.isNullOrEmpty()) {
+            throw IllegalArgumentException("빈문자열은 입력될 수 없습니다.")
+        }
```

Comment:
`require()`는 값을 만족하지 않으면 `IllegalArgumentException`을 발생시킵니다.

```suggestion
        require(!expression.isNullOrEmpty()) { "빈문자열은 입력될 수 없습니다." }
```